internal/docker/builder.go

```diff
@@ -179,12 +178,23 @@ func (d *Builder) ConstructBlueprintsIfNotExist(bs []b.Blueprint) error {
 }
 
 func (d *Builder) ConstructBlueprints(bs []b.Blueprint) error {
-	var bpWg sync.WaitGroup
-	bpWg.Add(len(bs))
+	errc := make(chan []error, len(bs))
 	for _, bprint := range bs {
-		go d.construct(bprint, &bpWg)
+		go (func(bprint b.Blueprint) {
+			errc <- d.construct(bprint)
+		})(bprint)
 	}
-	bpWg.Wait()
+	var errs []error
+	for i := 0; i < len(bs); i++ {
+		errs = append(errs, <-errc...)
```

Comment:
We don't ever close `errc` so we'll leak resources. Call `close(errc)` after this `for` loop.

Reply:
Done!